build/dist

```diff
@@ -184,7 +184,8 @@ echo "Kyuubi $VERSION $GITREVSTRING built for" > "$DISTDIR/RELEASE"
 echo "Java $JAVA_VERSION" >> "$DISTDIR/RELEASE"
 echo "Scala $SCALA_VERSION" >> "$DISTDIR/RELEASE"
 echo "Spark $SPARK_VERSION" >> "$DISTDIR/RELEASE"
-echo "Hadoop $SPARK_HADOOP_VERSION" >> "$DISTDIR/RELEASE"
+echo "Spark Hadoop $SPARK_HADOOP_VERSION" >> "$DISTDIR/RELEASE"
```

Comment:
do we rely on spark-hadoop to build for the compiler?

Reply:
yes, it actullay affect the built-in spark directory name, like `spark-3.1.2-hadoop2.7` the version is 2.7

Reply:
I mean the compiler, we don't rely on any hadoop jar from spark dist for compiling, right?

Reply:
that's right, it only used to help find the built-in SPARK_HOME